src/compose/ComposeBox.js

```diff
@@ -201,7 +209,8 @@ class ComposeBox extends PureComponent<Props, State> {
       isMessageFocused: false,
       isMenuExpanded: false,
     });
-    setTimeout(this.updateIsFocused, 200); // give a chance to the topic input to get the focus
+    // give a chance to the topic input to get the focus
+    this.messageBlurTimeoutId = setTimeout(this.updateIsFocused, 200);
```

Comment:
This still won't work: if two blur events are triggered in quick succession, this will get double-updated.

(Which isn't likely under ordinary circumstances – but if the phone's responding sluggishly for one reason or another, the user's inputs could be buffered – topic, topic, topic, try to select something near the start of the message, tap the topic again just in case, give up and start mashing the back button.)

Just adding a `clearTimeout` before the `setTimeout` should handle all cases, though. (And then there's no need for two separate timeout variables.)

Reply:
Ohhh yep, good catch, thanks a lot.

Reply:
OK, I've made that fix.